labs/query-enriched-event-data-with-spark/index.md

```diff
@@ -25,19 +25,56 @@ spark = SparkSession \
     .builder \
     .appName("Python Spark SQL") \
     .config("spark.sql.repl.eagerEval.enabled", True) \
+    .config("spark.sql.repl.eagerEval.truncate", 120) \
     .getOrCreate()
 ```
 
 ## Loading enriched event data
 
-We'll start by loading decision and conversion data from `/data`.
+The `OPTIMIZELY_DATA_DIR` environment variable may be used to specify the local directory where Enriched Event Data is stored.  If, for example, you've downloaded Enriched Event data and saved it in `optimizely_data` in your home directory, you can load that data in this notebook by executing the following command before launching Jupyter Lab:
+
+```
+$ export OPTIMIZELY_DATA_DIR=~/optimizely_data
+```
+
+If `OPTIMIZELY_DATA_DIR` is not set, data will be loaded from `./data` in your working directory.
 
 
 ```python
-spark.read.parquet("data/decisions").createOrReplaceTempView("decisions")
-spark.read.parquet("data/events").createOrReplaceTempView("events")
+import os
+
+base_data_dir = os.environ.get("OPTIMIZELY_DATA_DIR", "data")
+
+def read_data(path, view_name):
+    """Read parquet data from the supplied path and create a corresponding temporary view with Spark."""
+    print(f"Reading {view_name} data from {path}")
+    spark.read.parquet(path).createOrReplaceTempView(view_name)
 ```
 
+Enriched Event data is partitioned into two distinct datasets: [decisions](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification#decisions-2) and [conversions](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification#conversions-2).
+
+We'll load decision data from the `type=decisions` directory in the base data directory.
+
+
+```python
+decisions_dir = os.path.join(base_data_dir, "type=decisions")
+read_data(decisions_dir, "decisions")
+```
+
+    Reading decisions data from data/type=decisions
```

Comment:
Is this intended to be a header for the content above?

Reply:
No; this was `print` output from the `read_data` function.  I agree it looks confusing in the markdown file, so I removed it from the notebook.